src/Marten/Linq/MartenLinqQueryable.cs

```diff
@@ -47,6 +47,9 @@ public MartenLinqQueryable(IMartenSession session): base(new MartenLinqQueryProv
 
         public MartenLinqQueryProvider MartenProvider { get; }
 
+        // TODO: Convert this to property?
```

Comment:
Maybe, but don't fuss with that in this PR is my vote.

Reply:
Ok. I've updated it to look like other `TODO`s in the code. Let me know if it's worth leaving in.